stock_barcodes_gs1/tests/test_stock_barcodes_gs1.py

```diff
@@ -19,10 +22,10 @@ def setUp(self):
             "01993167101234533101002620130" "5041710ABC123214145354"
         )
 
-        self.product_wo_tracking_gs1 = self.product_wo_tracking.copy(
+        self.product_wo_tracking_gs1 = self.product_wo_tracking.with_context({}).copy(
```

Comment:
Why this ?

Reply:
In the test I am copying a product.product, if I do not reset context the product.template does not create the variant...
Is better solution (only for test that @Tardo applied in this case)